Detectors/DCS/src/DetectorsDCSLinkDef.h

```diff
@@ -0,0 +1,15 @@
+// In applying this license CERN does not waive the privileges and immunities
```

Comment:
the licence is cut

Reply:
Yes, sorry, I worked in a "funny" mode yesterday morning, and I did not notice.